### PR TITLE
[LIB-474] .npmignore

### PR DESCRIPTION
**Pull request by jakubbilko**

### CODE REVIEW

**Comment by jakubbilko** [comment]
@dancio I guess we don't really need both `src` and `lib` in the npm module (less files to download), but the tests use the files from `src`, so maybe I should modify them to use the compiles files from `lib`. What do you think?

**Comment by dancio** [comment]
imho we should have src because someone maybe will want to require es6 source and compile it by hand with own settings :)

File: .npmignore [hunk 1]
```diff
@@ -0,0 +1,10 @@
+node_modules
+coverage
+docs
+tutorials
+dist
+src
```
Comment (dancio) [inline .npmignore]: so imho we should attach src files

Comment (jakubbilko) [inline .npmignore]: :ok:

**Pull request merged**